### DISCUSSION TITLE
what class does usb host support in latest zephyr version?

### DISCUSSION THREAD

**Discussion by Dream-LL** [zephyrproject-rtos/zephyr#79161] [General] [answered]
i can not see the host support class  in zephyr document, 

only introduce usb device class like audio  acm  dfu and so on
https://docs.zephyrproject.org/latest/connectivity/usb/device/usb_device.html#supported-usb-classes

**Comment by henrikbrixandersen** [comment, marked as answer]
USB host support is currently experimental and very minimalistic in Zephyr.

**Comment by DerekSnell** [comment]
Hi @Dream-LL ,
This RFC https://github.com/zephyrproject-rtos/zephyr/issues/42066 tracks the plans of progress for Zephyr's USB Host stack.
